client/src/components/LinkTo.js

```diff
@@ -47,6 +47,8 @@ export default class LinkTo extends Component {
 
     target: PropTypes.string,
     whenUnspecified: PropTypes.string,
+    modelType: PropTypes.string,
+    model: PropTypes.object,
     style: PropTypes.object,
     ...modelPropTypes
```

Comment:
Why not use `model={…} modelType={…}` everywhere `LinkTo` is called instead of the (variable) prop named after the model? It would make the code cleaner (I never liked the fact that the props for `LinkTo` were not fixed/constant).

Reply:
I'll see if I can do a bulk-update myself.

Reply:
Done.